client.php

```diff
@@ -33,9 +33,53 @@
 } else {
     RespondWithError(400, 'No Story Requested');
 }
-?>
 
+$wordLinkFilePath =  __DIR__ . '/data/' . ($language ?: "en")     . '/wordlinks.csv';
+$handle = fopen($wordLinkFilePath, 'r');
+$wordLinkTerms = [];
+
+// parse header row and ignore data
+fgetcsv($handle);
+// Read the file line by line
+while (($row = fgetcsv($handle, 0, ",")) !== FALSE) {
+    // skip empty row if there is any
+    if (isset($row[1]) && !empty($row[1])) {
+        $alternateForms = "";
+        $backTranslations = "";
+        $relatedTerms = "";
+
+        if (!empty($row[2])) {
+            $alternateForms =  explode(",", $row[2]);
+        }
+
+        if (!empty($row[3])) {
+            $backTranslations =  explode(";", $row[3]);
+        }
+
+        if (!empty($row[5])) {
+            $relatedTerms =  explode(",", $row[5]);
+        }
+
+        $wordLinkTerms[strtolower(trim($row[1]))] = [
+            "alternateTerms" => $alternateForms,
+            // "backTranslations" => $backTranslations, // TODO waiting for confirmation
```

Comment:
Is this still TODO? If not, please remove. If so, please clarify what you're waiting for.